### PR TITLE
Add support for set graphNode's title font color

### LINKED ISSUE #82085: There was no way to set GraphNode's title font color
### Godot version

v4.2.dev5.official [e3e2528ba]

### System information

Windows 10, 64bit

### Issue description

There was no way to set GraphNode's title font color

### Steps to reproduce

/

### Minimal reproduction project

/

### PR DESCRIPTION
**Pull request by SaNeOr**
Close #82085

### CODE REVIEW

**Comment by SaNeOr** [comment]
@AThousandShips Looks like the documentation is missing. How should I add them?

**Comment by AThousandShips** [comment]
See [here](https://docs.godotengine.org/en/stable/contributing/documentation/updating_the_class_reference.html#updating-class-reference-when-working-on-the-engine)

File: doc/classes/GraphNode.xml [hunk 1]
```diff
@@ -254,6 +254,9 @@
 		<theme_item name="resizer_color" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 1)">
 			The color modulation applied to the resizer icon.
 		</theme_item>
+		<theme_item name="title_color" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 1)">
+			The color of title.
```
Comment (AThousandShips) [inline doc/classes/GraphNode.xml:258]:
```suggestion
			The color of the node title.
```

**Comment by YuriSizov** [comment]
The way to set GraphNode's title color is to use the new `GraphNodeTitleLabel` type which has the `font_color` property. You should be able to configure it in your theme resource.

**Pull request closed without merging**